Modules/Dining/MITDiningHouseMealListViewController.m

```diff
@@ -152,9 +157,17 @@ - (void)updateCurrentlyDisplayedItems
             }
         }
         self.currentlyDisplayedItems = filteredItems;
-        [self updateNotificationLabel];
     }
+    
+    if (self.currentlyDisplayedItems.count > 0) {
+        self.tableView.tableFooterView = nil;
+    } else {
+        self.tableView.tableFooterView = [UIView new];
```

Comment:
When there's meals, they want the normal tableview behavior where it shows empty cells.  If there's no meal, or no items, they want the tableview to be blank.  This makes those cells not show up.